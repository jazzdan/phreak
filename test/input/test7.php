<?php

class Foo {
    const HELLO = "world";
    public $foo;
}
