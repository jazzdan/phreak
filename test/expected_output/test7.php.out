class Foo
{
    public const HELLO = "world"
    public $foo;
}



